plugin/githubinator.vim

```diff
@@ -56,17 +56,15 @@ function! s:generate_url()
     " Change `:` to `/` (in case of ssh remote).
     let l:remote = system('git config remote.origin.url')
     let l:remote = substitute(l:remote, 'git@\|git:', 'https://', 'g')
-    let l:remote = substitute(l:remote, '\.git.$', '', '')
-    echom string(l:remote)
+    let l:remote = substitute(l:remote, '\.git.$\|\n', '', '')
```

Comment:
newline character at the tail of URL is deleted here.
`\n` is added in the regexp